project/Dependencies.scala

```diff
@@ -53,6 +52,7 @@ object Dependencies {
     "joda-time"            % "joda-time"         % "2.2" withJavadoc(),
     "org.joda"             % "joda-convert"      % "1.2",
     "org.lz4"              %  "lz4-java"         % "1.4",
+    "org.agrona"           %  "agrona"           % "0.9.35",
```

Comment:
Interesting.  Is this needed?  Many other things already pull in Agrona

Reply:
such as Akka

Reply:
Yes, this is needed. Earlier version of kamon directly pulled akka dependencies and that pulled agrona dependency. But with newer version, it does not.